doc/specs/wiki.md

```diff
@@ -0,0 +1,315 @@
+# Wiki Specs
+
+## Table of contents
+
+1. Motivation
+2. Business rules
+3. Use cases
+
+## Motivation
+
+The site contains a variety of users with a vast wealth of knowledge. The wiki is a place where users can put their knowledge to work, and write articles for users to use. These may be supplementary to other documents on the site, or standalone documents such as guides.
+
+## Business rules
+
+* Articles must be able to be read/write locked based on user class.
+* Users with the wiki administration permission must have full access to all articles.
+* Article creation must be associated with a permission.
+* Duplicate aliases, tags, and dedicated terms must be unique and not empty.
```

Comment:
Sentence implies that Duplicate aliases must not be empty, what if an article doesn't have any aliases?